packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -12,14 +13,13 @@ const selectors = {
 
 	// Results
 	resultsPlaceholder: '.inline-help__results-placeholder-item',
-	resultsList: '.inline-help__results',
-	results: '.inline-help__results-item',
+	results: ( category: string ) => `[aria-labelledby="${ category }"] .inline-help__results-item`,
```

Comment:
The category here refers to either the `support ariticle` or `how-to` result types that are shown for every valid search query.